.vib/spring-cloud-skipper/goss/spring-cloud-skipper.yaml

```diff
@@ -0,0 +1,14 @@
+command:
+  check-spring-cloud-skipper:
+    timeout: 20000
+    exec: timeout --preserve-status 15 /opt/bitnami/java/bin/java -jar /opt/bitnami/spring-cloud-skipper/spring-cloud-skipper.jar
```

Comment:
Same feedback as in https://github.com/bitnami/containers/pull/34286/files#r1200343976